xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAESink.cpp

```diff
@@ -661,33 +681,48 @@ void CActiveAESink::Process()
   }
 }
 
-void CActiveAESink::EnumerateSinkList(bool force)
+void CActiveAESink::EnumerateSinkList(bool force, std::string driver)
 {
   if (!m_sinkInfoList.empty() && !force)
     return;
 
   if (!CAESinkFactory::HasSinks())
     return;
 
+  std::vector<AE::AESinkInfo> tmpList(m_sinkInfoList);
+
   unsigned int c_retry = 4;
   m_sinkInfoList.clear();
-  CAESinkFactory::EnumerateEx(m_sinkInfoList, false);
+
+  if (!driver.empty())
+  {
+    for (auto const &info : tmpList)
+    {
+      if (info.m_sinkName != driver)
+        m_sinkInfoList.push_back(info);
+    }
+  }
+
+  CAESinkFactory::EnumerateEx(m_sinkInfoList, false, driver);
   while (m_sinkInfoList.empty() && c_retry > 0)
   {
     CLog::Log(LOGNOTICE, "No Devices found - retry: %d", c_retry);
     Sleep(1500);
     c_retry--;
     // retry the enumeration
-    CAESinkFactory::EnumerateEx(m_sinkInfoList, true);
+    CAESinkFactory::EnumerateEx(m_sinkInfoList, true, driver);
   }
   CLog::Log(LOGNOTICE, "Found %lu Lists of Devices", m_sinkInfoList.size());
-  PrintSinks();
+  PrintSinks(driver);
 }
 
-void CActiveAESink::PrintSinks()
+void CActiveAESink::PrintSinks(std::string &driver)
```

Comment:
> Left-align `*` and `&` to the base type they modify.

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#52-pointer-and-reference-types